contents/dataframesmeta_transform.md

```diff
@@ -0,0 +1,87 @@
+## Column Transformation {#sec:dataframesmeta_transform}
+
+Whereas the `@select` macro variants performs column selection,
+the `@transform` macro variants do _not_ perform any column selection.
+It can either _overwrite_ existent columns or _create_ new columns that will be added to the right of our `DataFrame`.
+
+For example, the previous operation on `:grade` can be invoked as a transformation with:
+
+```jl
+sco("""
+# rtransform 1 # hide
+@rtransform df :grade_100 = :grade * 10
+"""; process=without_caption_label
+)
+```
+
+As you can see, `@transform` does not perform column selection,
+and the `:grade_100` column is created as a new column and added to the right of our `DataFrame`.
+
+`DataFramesMeta.jl` macros also support `begin ... end` statements.
+For example, suppose that you are creating two columns in a `@transform` macro:
+
+```jl
+sco("""
+# rtransform 2 # hide
+@rtransform df :grade_100 = :grade * 10 :grade_5 = :grade / 2
+"""; process=without_caption_label
+)
+```
+
+It can be cumbersome and difficult to read the performed transformations.
+To facilitate that, we can use `begin ... end` statements and put one transformation per line:
+
+```jl
+sco("""
+# rtransform 3 # hide
+@rtransform df begin
+    :grade_100 = :grade * 10
+    :grade_5 = :grade / 2
+end
+"""; process=without_caption_label
+)
+```
+
+This makes much easier to analyze code.
```

Comment:
```suggestion
```
This sentence is already clear when reading the two sentences above plus the code and the output.